guides/code-signing/code-signing-macos.md

```diff
@@ -127,9 +193,9 @@ There are two mandatory fields for `osxNotarize` if you are using this strategy:
 
 | Field             | Type   | Description                                                                                                                                                                                                   |
 | ----------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `appleId`         | string | Apple ID associated with your Apple Developer account                                                                                                                                                         |
-| `appleIdPassword` | string | App-specific password                                                                                                                                                                                         |
-| `teamId`          | string | The Apple Team ID you want to notarize under. You can find Team IDs for team you belong to by going to [`https://developer.apple.com/account/#/membership`](https://developer.apple.com/account/#/membership) |
+| `appleId`         | string | Usually the email address you used to create your Apple Developer account.                                                                                                                                                         |
+| `appleIdPassword` | string | A one-time password that can be create via the Apple Developer website.                                                                                                                                                                                       |
+| `teamId`          | string | The Apple Team ID you want to notarize under. It is the set of characters inside the brackets at the end of your identity name. |
```

Comment:
The old descriptions were more helpful and used the terminology as used by Apple.